ql/src/java/org/apache/hadoop/hive/ql/plan/ExprNodeDescUtils.java

```diff
@@ -1136,4 +1173,30 @@ public static boolean isOr(ExprNodeDesc expr) {
     return false;
   }
 
+  public static boolean isAnd(ExprNodeDesc expr) {
+    if (expr instanceof ExprNodeGenericFuncDesc) {
```

Comment:
I think you could use `ExprNodeDescExprFactory.isANDFuncCallExpr` or `FunctionRegistry.isOpAnd(expr)`?